src/cacheLink.ts

```diff
@@ -0,0 +1,44 @@
+import {
```

Comment:
This file is unused - can we move it into the tests, or an unused folder, or remove it?